runners/apex/src/test/java/org/apache/beam/runners/apex/ApexRunnerTest.java

```diff
@@ -82,7 +82,7 @@ public void testParDoChaining() throws Exception {
     ApexPipelineOptions options = PipelineOptionsFactory.as(ApexPipelineOptions.class);
     DAG dag = TestApexRunner.translate(p, options);
 
-    String[] expectedThreadLocal = {"/CreateActual/FilterActuals/Window.Assign"};
+    String[] expectedThreadLocal = {"/GroupGlobally/RewindowActuals/Window.Assign"};
```

Comment:
It might just be better to not expect some hard-coded operator name here, but I'm not really sure what exactly this is intending to test.

Reply:
From the name of the test, I think this is a hacked way to ensure that the transforms are actually fused into a single Apex operator. Your adjustment LGTM.